src/main/java/com/powsybl/openloadflow/network/util/ActivePowerDistribution.java

```diff
@@ -28,7 +29,7 @@ public interface Step {
 
         String getElementType();
 
-        List<ParticipatingElement> getParticipatingElements(Collection<LfBus> buses);
+        List<ParticipatingElement> getParticipatingElements(Stream<LfBus> buses);
```

Comment:
I think (to confirm) this is a bad practice to pass a stream as a method parameter as once iterated it cannot be reused anymore.